TS01-Time-Series_Metadata.Rmd

```diff
@@ -65,37 +60,20 @@ refresher on setting the working directory, please view the <a href="XXX" target
 
 </div>
 
-#Understand Your Data 
-There are many different contexts for time series data in ecology.  We will be 
-exploring the relationships between air temperature, precipiation,  
-photosyntheically active radiation (PAR) and plant phenology,especially the 
-greening up and browning down temporal patterns 
-
-When initiating a new script and workflow it is always a good idea to 
```

Comment:
Is this covered elsewhere?  I didn't think it was.  This section was originally added to meet our desires to teach good coding skills not just the techniques.  I just want to make sure it is covered.